pydicom/jsonrep.py

```diff
@@ -41,3 +47,202 @@ def convert_to_python_number(value, vr):
         else:
             value = number_type(value)
     return value
+
+
+class JsonDataElementConverter(object):
+    """Handles conversion between JSON struct and :class:`DataElement`."""
+
+    def __init__(self, dataset_class, tag, vr, value, value_key,
+                 bulk_data_uri_handler):
+        """Create a new converter instance.
+
+        Parameters
+        ----------
+        dataset_class : Dataset derived class
```

Comment:
Should be `dataset_class : dataset.Dataset derived class`, I think

Reply:
Ok.